### DISCUSSION TITLE
Dokku Pro demo?

### DISCUSSION THREAD

**Discussion by gregdev** [dokku/dokku#5165] [General]
Is it possible to install Dokku Pro on a dev server to test it out before purchasing? I'm currently trying to work out the best way to move 100+ low-traffic WordPress sites from WHM over to a Docker-based setup and Dokku Pro could be the go, but I'd like to test it out to make sure before I ask my boss if I can spend $1000 (stoopid exchange rate).

Cheers

**Comment by robertoandrade** [comment]
Same here, any info on it?

**Comment by gamedevsam** [comment]
Not currently as far as I understand it. Due to the way the dokku-pro licensing works, once the binary is made available piracy is not hard to do. It's important for dokku team to be careful about giving access to the binary only to loyal paying customers.

I think better screenshots and a video would go a long way to help sell dokku-pro, but a demo is difficult to setup without spending a lot of time on limits and restricting what demo users might be able to do.
